sia/hostdb/hostdb.go

```diff
@@ -84,14 +114,16 @@ func (hdb *HostDatabase) Update(initialStateHeight consensus.BlockHeight, rewoun
 
 	// Add hosts found in blocks that were applied.
 	for _, b := range appliedBlocks {
-		var entries []HostEntry
+		var entries []components.HostEntry
 		entries, err = findHostAnnouncements(initialStateHeight, b)
 		if err != nil {
 			return
 		}
 
 		for _, entry := range entries {
+			hdb.unlock()
 			err = hdb.Insert(entry)
+			hdb.lock()
```

Comment:
I actually did that in multiple other places, this one must have slipped thorugh